### DISCUSSION TITLE
Ability to show both count and log lines in Logs visualization

### DISCUSSION THREAD

**Discussion by mattparkes** [grafana/grafana#59448] [Dashboards, Panels & Visualizations]
Similiar to [Discussion #59446](https://github.com/grafana/grafana/discussions/59446), in a Log visualization it would be great to be able to simultaneously see the `count_over_time` _and_ the actual logs themselves, without having to have 2 panels.

This would be similar to what the Explore tab already does.

Currently we're having to replicate this by duplicating all our panels and making a second `count_over_time()` panel.

I understand this awkwardly combines Logs and Time Series in one panel, so perhaps this should be a completely separate visualization?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
